### ISSUE TITLE
Cannot see some pictures

### ISSUE THREAD

**Issue by Buco7854** [varun-raj/immich-power-tools#7]
My API Key use m'y admin account Key.

I have lots of potential albums suggested but in some case thé pictures can't be displayed when the pictures isnt "own" by same account.

Should power tools bé used for single user or is it actually a bug.

**Comment by varun-raj** [comment]
Hi @Buco7854 Yes, that's right I completly forgot about as I use it solo. Will implement the account filtering today and push it in next release

**Comment by Buco7854** [comment]
> Hi @Buco7854 Yes, that's right I completly forgot about as I use it solo. Will implement the account filtering today and push it in next release

Looking forward to it, thanks!

**Comment by varun-raj** [comment]
Hi @Buco7854

This is fixed in the last docker release. Can you please check it now and let me know?

**Comment by Buco7854** [comment]
Yes, seems like it is fixed. The people and albums are only mine. There is still some blank images in albums tho, I would need to check why but I'll probably not be able to check it today and tomorrow.

**Comment by Buco7854** [comment]
Seems like some pictures doesn't load cause immich proxy returns a 500 and it seems like the request to the thumbnail was answered with a 404 by immich.
